<launch>

    <!-- Landmark Manager Args -->
    <arg name="landmark_config"         default="$(find interbotix_landmark_modules)/landmarks/landmarks.yaml"/>
    <arg name="fixed_frame"             default="landmarks"/>
    <arg name="obs_frame"               default="camera_color_optical_frame"/>
    <arg name="standalone_tags"         default="$(find interbotix_perception_modules)/config/tags.yaml"/>

    <!-- Landmark Manager Node -->
    <node
        pkg="interbotix_landmark_modules"
        type="landmark_manager"
        name="landmark_manager"
        required="true"
        output="screen">
        <param name="landmark_config"   value="$(arg landmark_config)"/>
        <param name="fixed_frame"       value="$(arg fixed_frame)"/>
        <param name="obs_frame"         value="$(arg obs_frame)"/>
        <rosparam command="load"        file="$(arg standalone_tags)"/>
    </node>

</launch>
